File: speedTests/PrimesNumber/code.js
function isPrime(n){

    for (let i=2;i<=n/2; i++){
        if (!(n%i))return 0;
    }
    return 1;
}

let numPrimes=0;
for (let i={{min}};i<{{max}}; i++)
numPrimes += isPrime(i);

console.log(numPrimes);

File: speedTests/PrimesNumber/code.py
def isPrime(n):
    for i in range(2,n//2+1):
        if (not (n%i)):
            return 0
    return 1

numPrimes=1

for i in range ({{min}},{{max}}):
  numPrimes+=isPrime(i)
print(str(numPrimes))